### PR TITLE
Blog - How to Automate Common Tasks with Shell Scripts

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks.

### CODE REVIEW

**Comment by msali123** [comment]
@jalletto build failing

**Comment by msali123** [comment]
@balapriyac [Preview Link](https://6470797ce782c0710ef47d33--ephemeral-narwhal-3ac3b2.netlify.app/blog/automate-tasks-shell-scripts/)

File: blog/_posts/2023-06-23-automate-tasks-shell-scripts.md [hunk 1]
```diff
@@ -0,0 +1,668 @@
...
+
+1. Create a file with the  `.sh` extension in your terminal and open it using any text editor such as [Vim](https://www.vim.org/) or [Nano](https://www.nano-editor.org/). For example, you can call this file `backup.sh`:
+
+   ~~~{.bash caption=">_"}
+      vim backup.sh
+   ~~~
+
+2. Define the source and destination variables:
+
+   ~~~{ caption="backup.sh"}
```
Comment (jalletto) [inline blog/_posts/2023-06-23-automate-tasks-shell-scripts.md:94]:
```suggestion
   ~~~{.bash caption="backup.sh"}
```

Comment (jalletto) [reply blog/_posts/2023-06-23-automate-tasks-shell-scripts.md:94]: all these bash ones needs to have `.bash`

File: blog/_posts/2023-06-23-automate-tasks-shell-scripts.md [hunk 2]
```diff
@@ -0,0 +1,668 @@
...
+
+internal-links:
+ - Automate
+ - Shell Scripts
+ - Autamating Tasks
+ - Bash
+ - Shell Scripts
+---
+
+**We're [Earthly.dev](https://earthly.dev/). We make building software simpler and therefore faster using containerization. This article is about how to automate common tasks with shell scripts but if you're interested in a different approach to building and packaging software then [check us out](/).**
```
Comment (jalletto) [inline blog/_posts/2023-06-23-automate-tasks-shell-scripts.md:17]:
```suggestion
**We're [Earthly.dev](https://earthly.dev/). We make building software simpler and therefore faster using containerization. This article is about how to automate common tasks with shell scripts. If you're interested in a different approach to building and packaging software then [check us out](/).**
```

**Review by jalletto — CHANGES_REQUESTED** [review]

**Comment by msali123** [comment]
@jalletto [Changes Done](https://6478fd70f69d8c0fd493590a--ephemeral-narwhal-3ac3b2.netlify.app/blog/automate-tasks-shell-scripts/)

**Review by jalletto — APPROVED** [review]

**Pull request merged**